mrtt-ui/src/language.js

```diff
@@ -2,14 +2,16 @@ const error = {
   apiLoad: 'Loading data from the api failed. Please try again.',
   delete: 'Deleting that item failed. Please try again.',
   getItemDoesntExistMessage: (item) => `That ${item} doesnt exist.`,
+  generic: 'Something went wrong. Please try again.',
   submit: 'Saving failed. Please try again.'
 }
 
 const success = {
-  submit: 'The data has been saved.',
-  getEditThingSuccessMessage: (thing) => `${thing} has been edited.`,
   getCreateThingSuccessMessage: (thing) => `${thing}, has been created.`,
-  getDeleteThingSuccessMessage: (thing) => `${thing} has been deleted.`
+  getDeleteThingSuccessMessage: (thing) => `${thing} has been deleted.`,
```

Comment:
what is thing? can you be more specific?

Reply:
I just added the comma. But thing is the thing/noun being deleted

Reply:
agree about submitting state, but thats scope creep for this ticket. Feel free to make another ticket and we can priortize it as a team in the next meeting.

Reply:
[added issue here](https://github.com/globalmangrovewatch/gmw-users/issues/308)